File: backends/php-laravel/app/Domain/Client/ValueObjects/ClientQrVo.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\Domain\Client\ValueObjects;

use App\Support\Traits\Getter;
use App\Support\ValueObjects\AbstractValueObject;

/**
 * スマホアプリ連携用QRコードValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Client\ValueObjects
 *
 * @method string|null getIdentifier()
 * @method string|null getDeeplinkUrl()
 */
class ClientQrVo extends AbstractValueObject
{
    use Getter;

    protected ?string $identifier = null;
    protected ?string $deeplinkUrl = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/client/valueobjects/ClientQrVo.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.client.valueobjects;

import com.authorization.support.valueobjects.AbstractValueObject;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * スマホアプリ連携用QRコードValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class ClientQrVo extends AbstractValueObject {

    private String identifier;
    private String deeplinkUrl;
}
